src/pages/UserHome.jsx

```diff
@@ -81,27 +95,27 @@ export default function UserHome() {
           </Button>
         </Grid>
       </Grid>
-      <center>
+      <Box className={classes.centerIcon}>
+        <InboxIcon
+          style={{ fontSize: 200 }}
+          className={classes.addIconSpacing}
+        />
+      </Box>
+      <Box className={classes.centerTextAndLink}>
         <Typography
           variant="h5"
           component="h2"
-          gutterBotton
+          gutterBottom
           className={classes.noScholarshipsAdded}>
           No Scholarships Added Yet
         </Typography>
-      </center>
-      <center>
-        <Typography
-          variant="h10"
-          component="h2"
-          gutterButton
-          color="primary"
-          className={classes.addScholarship}>
-          <p>
-            <a href="more-scholarships">Add A Scholarship</a>
-          </p>
+
+        <Typography variant="h10" component="h2" gutterButton color="primary">
```

Comment:
*`gutterBottom`, though in this case it might not be needed since there isn't anything underneath.